http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -986,6 +986,9 @@ definitions:
         items:
           type: string
         description: The scopes for the cluster.
+      customClusterEnvironmentVariables:
```

Comment:
is this change related to this PR?

Reply:
This field was added in a previous PR, but I noticed it was not added to swagger. I also had to persist it in the database in this PR, since back Leo is doing the cluster creation.